src/main/java/com/openlattice/chronicle/services/ChronicleService.java

```diff
@@ -51,9 +52,9 @@ Integer logData(
 
     boolean isKnownParticipant( UUID studyId, String participantId );
 
-    void deleteParticipantAndAllNeighbors( UUID studyId, String participantId );
+    Void deleteParticipantAndAllNeighbors( UUID studyId, String participantId, DeleteType deleteType, String token );
 
-    void deleteStudyAndAllNeighbors( UUID studyId );
+    Void deleteStudyAndAllNeighbors( UUID studyId, DeleteType deleteType, String token );
```

Comment:
also going to want `void` on these two

Reply:
Okay, actually in this case you *do* want `void` because this isn't (as far as I can tell) being called by retrofit